evaluate/evaluate_test.go

```diff
@@ -121,10 +121,12 @@ func TestEvaluate(t *testing.T) {
 			// Normalize assessments.
 			for _, ls := range actualAssessments {
 				for _, rs := range ls {
-					for _, a := range rs {
-						if v, ok := a[metrics.AssessmentKeyProcessingTime]; ok {
-							if assert.Greater(t, v, uint64(0)) {
-								delete(a, metrics.AssessmentKeyProcessingTime)
+					for _, ts := range rs {
+						for _, a := range ts {
+							if v, ok := a[metrics.AssessmentKeyProcessingTime]; ok {
+								if assert.Greater(t, v, uint64(0)) {
+									delete(a, metrics.AssessmentKeyProcessingTime)
+								}
```

Comment:
Out of context.

We already had a logic that ignores all nondeterministic parts of assessments. So with https://github.com/symflower/eval-dev-quality/commit/b207e284f4a76054dd3e2bf4259f7d2e744bffe7 we now do that twice in two separate locations. Could you please either resolve this or keep it as a follow-up?

Reply:
Not sure what you mean. If you are talking about the "normalize assessments, then this is the very same location.

Reply:
It is... sorry for not clarifying. There already is `metricstesting.AssertAssessmentsEqual`. And now this as well.

Reply:
its not a big deal... just good to keep an eye out in the future

Reply:
Briefly looked at this. But I don't want to touch this right now. Might be that some tests are failing again, and this would delay merging the PR.

Reply:
Added to #169